### PR TITLE
Refactor request response

### PR DESCRIPTION
**Pull request by shubham-y**
Updated the request and response json keys to follow the CamelCase naming convention.

### CODE REVIEW

File: createUserFeatureFlag/main.go [hunk 1]
```diff
@@ -83,6 +83,8 @@ func handler(req events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse,
 		Status:    strings.ToUpper(requestBody.Status),
 		CreatedAt: time.Now().Unix(),
 		CreatedBy: requestBody.UserId,
+		UpdatedAt: time.Now().Unix(),
```
Comment (vikhyat187) [inline createUserFeatureFlag/main.go:86]: was this not being updated earlier? @shubham-y

File: getUserFeatureFlag/main.go [hunk 2]
```diff
@@ -15,7 +14,7 @@ import (
 	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
 )
 
-func processGetById(userId string, flagId string) (*models.FeatureFlagUserMapping, error) {
+func processGetById(userId string, flagId string) (*utils.FeatureFlagUserMappingResponse, error) {
 
 	db := database.CreateDynamoDB()
```
Comment (vikhyat187) [inline getUserFeatureFlag/main.go:20]: can you comment near line no 24 that `userId`  is the partition key and `flagId` is the sort key

**Review by vikhyat187 — COMMENTED** [review]
few minor comments

File: createUserFeatureFlag/main.go (hunk 1 above)
Comment (shubham-y) [reply createUserFeatureFlag/main.go:86]: Yes, previously I missed it.

File: getUserFeatureFlag/main.go (hunk 2 above)
Comment (shubham-y) [reply getUserFeatureFlag/main.go:20]: Done

**Review by vikhyat187 — APPROVED** [review]

**Pull request merged**